### PR TITLE
Fix Mobile Footer text aligment

### PR DESCRIPTION
**Pull request by dimlaksana**
## Checklist

- [x] Fixing bug on Mobile Footer
- [x] Github action build passing
- [x] No merge conflicts

### CODE REVIEW

File: src/css/components/_footer.scss [hunk 1]
```diff
@@ -89,9 +89,28 @@
...
+    left: 0;
+  }
+
+  .footer .madeWith {
+    position: relative;
+    right: 0px;
+    order: 2;
+    justify-content: center;
+  }
+}
```
Comment (subeshb1) [inline src/css/components/_footer.scss:116]:
```suggestion
}

```

**Comment by subeshb1** [comment]
Thanks for your contribution! @dimlaksana  Can you also add the screenshot?

**Comment by dimlaksana** [comment]
here it is 
Screenshot_20201017_004927

**Comment by subeshb1** [comment]
Great @dimlaksana! Good job

**Review by subeshb1 — APPROVED** [review]

**Pull request merged**